websites/A/AnimeTvOnline/metadata.json

```
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "521762564062052393",
+		"name": "Dwifte#0661"
+	},
+	"service": "AnimeTvOnline",
+	"description": {
+		"en": "An Italian site to watch anime online."
+	},
+	"url": "animetvonline.org",
```

Site supports `www` subdomain.
```suggestion
	"url": ["www.animetvonline.org", "animetvonline.org"],
```